programming/index.md

```diff
@@ -21,3 +21,11 @@ Our code is versioned using [Git](https://git-scm.com/). Git lets us collaborate
 
 - [Brief Introduction](https://try.github.io)
 - [Branching and Code Review](branching)
+
+## Unit Testing
+We use unit testing to help catch bugs in our code. This lets us quickly identify what parts of our code need to be fixed.
+
+- [What Is Unit Testing?](https://code.tutsplus.com/articles/the-beginners-guide-to-unit-testing-what-is-unit-testing--wp-25728)
+- [Unit Testing in Python](https://jeffknupp.com/blog/2013/12/09/improve-your-python-understanding-unit-testing/)
+- [`unittest`](https://docs.python.org/3/library/unittest.html)
```

Comment:
@BrendanBurkhart I believe we are using `py.test`, not `unittest`.

Reply:
Have we made a decision? If possible, I would like to advocate for `unittest`. It is a standard module, and is easier to use, especially for new comers to unit testing. I haven't used it all that much, but I think`py.test` tends to make it much harder to structure unit tests nicely.

Reply:
I believe pyfrc is pretty tied to py.test

Reply:
http://robotpy.readthedocs.io/projects/pyfrc/en/stable/custom.html

Reply:
😞

Reply:
Ok, in that case all those links definitely need to be replaced.